azkaban-common/src/main/java/azkaban/storage/StorageConfig.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ */
+
+package azkaban.storage;
+
+import azkaban.Constants;
+import azkaban.utils.Props;
+import com.google.inject.Inject;
+
+
+public class StorageConfig {
+
+  private final Props props;
```

Comment:
We already have `Props` for that and it is not easy to manage. I personally prefer modular configs since they are much smaller and easier to deal with and also address a specific context. However, this class is expected to go through some churn.